listenbrainz/webserver/static/js/src/recommendations/Recommendations.tsx

```diff
@@ -159,6 +235,7 @@ export default class Recommendations extends React.Component<
   };
 
   afterRecommendationsDisplay() {
+    this.loadFeedback();
```

Comment:
We're not going to need this, it was a bad suggestion from my part !

Reply:
Change getFeedback to fetch feedback for the recordings in the state rather than in the props (i.e the current page the user sees instead of for all recs):

```
getFeedback = async () => {
    const { recommendations } = this.state;
    const { user } = this.props;
```

should revert this too?

Reply:
Yes, that also needs to be reverted; You were absolutely right the first time and we only need to get feedback once, and for all the recommendations.